source-map.bs

```diff
@@ -362,7 +368,7 @@ also accept `//@` but `//#` is preferred.
 
 This recommendation works well for JavaScript, it is expected that other source files will
 have different conventions.  For instance, for CSS `/*# sourceMappingURL=<url> */` is proposed.
-On the WebAssembly side, such URL should be placed as a UTF-8 encoded string in the custom section ([[WasmCustomSection]]) named `sourceMappingURL`. 
+On the WebAssembly side, such a URL is encoded using [[WasmNamesBinaryFormat]], and it's placed in the custom section ([[WasmCustomSection]]) named `sourceMappingURL`.
```

Comment:
s/placed in/placed as/ (to express that this is the totality of the custom section, not part of it)